wiki/Gameplay/Multiplayer/en.md

```diff
@@ -1,434 +1,380 @@
+The [grade](/wiki/Gameplay/Grade) for the play is placed behind the result like a watermark. Other players' results can be checked by clicking on their panel on the leaderboard.
 
-When this mode was selected, all players in the Match Setup will receive either red or blue flag.
-The players can click on it to change from blue to red or vice versa.
+Each player will receive their match result as a private message in the `#userlog` tab in [chat](/wiki/Client/Interface/Chat_console).
 
-It is entirely possible to start a match with one colour team only, and there is no handicap bonus for the other team with lesser members.
-Placement of the players prior to the match does not matter; all blue members will be grouped to blue team and all red members will be grouped to red team.
-There is no role (like "Leader" or "Captain") distinction in the team; just compete for both the team top spot and beating the other team score in total as a team.
+#### Team Vs
 
-The surviving highest scoring member of either team will be shown as `1st` at their respective team.
-Members in failed state will be located at the bottom of the team leaderboard and will not contribute to the team total as long as the members are in failed state.
+##### Interface
 
-#### Result
+![](img/Multi_team_vs.jpg "Team Vs interface")
 
-![](img/Multi_team_grade.jpg "Team Vs grade screen")
+`Team Vs` pits two teams, Blue and Red, against each other. Fulfill the win condition as closely as possible to reach the top spot in the team and compete with the other team.
 
-In the results screen, declaration of match winner will be based on team total and announced at the centre in winner's colour, with blue team total at the left and red team total at the right.
+Blue is on the left and Red is on the right. A crown at the centre moves left and right to show which team is currently winning.
 
-**Only surviving members of the team's score will be counted**; it is possible for red team to be declared winner of the match when blue team has the superiority *just because* some blue team members providing the superiority were in failed state or all members in the blue team failed.
+Changing teams can be done while in the lobby. There is no handicap for teams with fewer members, and it is entirely possible to start a match with only one team.
 
-The `Total` tab will show the total amount of hit score (300, 100, etc) done by the whole team and the average of the accuracy of the team.
-Combo will not be counted in the team's `Total` tab.
-Grade will not be shown on `Total` tab but will be shown at player's result screen as watermark (if passed).
+##### Result
 
-### Tag Coop / Tag Team Vs
+![](img/Multi_team_grade.jpg "Team Vs results screen")
 
-#### Interface
+The winning team is announced at the top of the results screen.
 
-*Only available in osu! mode. **UNRANKED***
+Only players who are alive by the end of the match count towards their teams' totals.
```

Comment:
i've tentatively reworded this from "counting to team score", but i'm not sure if that's the case for the accuracy and combo win conditions as well